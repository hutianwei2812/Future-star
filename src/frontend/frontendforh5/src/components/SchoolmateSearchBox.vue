<template>
  <div class="searchbox">
    <div class="inputlabel">
      <input type="text" class="StInput" id="searchbox" placeholder="  按 关 键 词 搜 索 校 友" />
      <button class="s_button" onclick="getsearchcontent()">
        <i class="Si" id="Si"></i>
      </button>
    </div>
  </div>
</template>

<script>
export default {
  name: 'SchoolmateSearchBox',
  props: {}
}
</script>

<style scoped>
.StInput {
  width: 400px;
  height: 60px;
  border: 1 none black;
  font-size: 30px;
  background: 100 100;
  position: relative;
  margin: 5% 30% 2%;
}
.inputlabel {
  width: 1140px;
  position: relative;
  margin: auto;
}
.Si {
  position: absolute;
  top: 50%;
  right: 11px;
  margin-top: -8px;
  width: 16px;
  height: 16px;
  background-image: url(../assets/icon.png);
}
#Si {
  background-position: 0 -40px;
}
.s_button {
  position: relative;
  top: 5px;
  right: 390px;
  border: 0 none;
  width: 38px;
  height: 35px;
  overflow: visible;
  background: 0 0;
}
button {
  cursor: pointer;
}
</style>