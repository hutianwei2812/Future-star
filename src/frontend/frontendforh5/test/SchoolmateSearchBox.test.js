import { mount } from '@vue/test-utils'
import SchoolmateSearchBox from '@/components/SchoolmateSearchBox'
import Vue from 'vue'
import ElementUI from 'element-ui'
Vue.use(ElementUI)

test('renders correctly', () => {
  const wrapper = mount(SchoolmateSearchBox)
  expect(wrapper.element).toMatchSnapshot()
})