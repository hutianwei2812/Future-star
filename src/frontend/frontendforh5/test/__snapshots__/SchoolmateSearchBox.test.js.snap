// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  class="searchbox"
>
  <div
    class="inputlabel"
  >
    <input
      class="s_t_input"
      id="searchbox"
      placeholder="  按 关 键 词 搜 索 校 友"
      type="text"
    />
     
    <button
      class="s_button"
      onclick="getsearchcontent()"
    >
      <i
        class="s_i"
        id="s_i"
      />
    </button>
  </div>
</div>
`;
